examples/basic-tracer-node/index.js

```diff
@@ -12,7 +12,7 @@ provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
 provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
 
 // Initialize the OpenTelemetry APIs to use the BasicTracerProvider bindings
-opentelemetry.trace.setGlobalTracerProvider(provider);
+provider.register();
```

Comment:
Maybe, add a comment about why do we need to register or what happens behind the scene.